addons/l10n_it/migrations/0.8/pre-migrate.py

```diff
@@ -0,0 +1,11 @@
+def migrate(cr, version):
+
+    cr.execute("""
+        UPDATE account_tax
+           SET l10n_it_exempt_reason = CASE
+               WHEN l10n_it_exempt_reason = 'N2' THEN 'N2.2'
+               WHEN l10n_it_exempt_reason = 'N3' THEN 'N3.2'
```

Comment:
How do we know the former "generic" `N3` maps to the "intra-community" `N3.2` now?
Wouldn't `N3.6` be more suitable, since we don't/can't really know?